torch/autograd/profiler.py

```diff
@@ -252,14 +251,17 @@ def __init__(
             self.kineto_activities.add(ProfilerActivity.MTIA)
 
         self.profiler_kind = ProfilerState.KINETO
-        if self.use_cuda:
+        if self.use_device == "cuda":
```

Comment:
as @aaronenyeshi  suggested please keep use_cuda
```
if self.use_cuda or self.use_device == "cuda"
```

Reply:
Sure. Thank you.